invenio_communities/config.py

```diff
@@ -65,3 +66,37 @@
     ),
 }
 """Community roles."""
+
+COMMUNITIES_SEARCH_REQUESTS = {
```

Comment:
We have to come up with good name spacing here because there will be 4/5 search pages.
Can you please prepare a list of all the search pages and propose namespacing? Then, we can discuss it together.

Reply:
Communities records = COMMUNITIES_SEARCH_RECORDS
All communities = COMMUNITIES_SEARCH
Community requests = COMMUNITIES_SEARCH_REQUESTS
Community members = COMMUNITIES_SEARCH_MEMBERS
Community invitations = COMMUNITIES_SEARCH_INVITATIONS